fix: full text search

Ketcap opened the pull request:
Fixed  markdown on full text search.

janpio commented:
I think line 87/8 needs to be changed to triple backtick for this to work.
(You can make the newly added one and the one 2 lines before that also only triple backtick by the way. I see no need why it need to be 4 backticks here)

Ketcap commented:
It stops there for that section. And i can fix that for that sql line as well 👍

Ketcap commented:
@janpio I've fixed 2 more places to render properly.
- [x] Prisma code example fixed
- [x] Terminal code example fixed
- [x] Sql code example fixed
- [x] Page render issue fixed by missing " ` "

janpio commented on content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx:
````diff
@@ -18,24 +18,24 @@ The full-text Search API is still in **Preview**. You can enable `fullTextSearch
-    }
-    ```
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["fullTextSearch"]
+}
+```
 
 2: Generate the Prisma Client:
 
-    ```terminal copy
-    npx prisma generate
-    ```
+```terminal copy
+  npx prisma generate
+```
````
I think a better solution here would be to keep the indentation of the code blocks, and replace `1:` and  `2:` with `1.` and `2.` to make it a numbered list. Then this would look fine.

janpio commented:
Awesome. I think one of those can be fixed in a better way even - left a comment with the idea.

Then this would be ready to merge I would say.

Ketcap commented:
@janpio I've updated as you recommended looking good i've also fixed `Admonition` text rendering for bold in the page let me know if it needs more adjustment 👍

janpio reviewed (approved):
Great work, thank you!

molebox reviewed (approved):
![looks-good](https://user-images.githubusercontent.com/22930449/132180545-bae4c42d-8d96-45e6-b621-e6ee8eaec082.gif)
